MeshLib/Elements/CellRule.cpp

```diff
@@ -0,0 +1,41 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2015, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#include "CellRule.h"
+
+#include "logog/include/logog.hpp"
+
+#include "GeoLib/AnalyticalGeometry.h"
+
+#include "MeshLib/Node.h"
+#include "Face.h"
+
+namespace MeshLib
+{
+
+const unsigned CellRule::dimension;
+
+bool CellRule::testElementNodeOrder(const Element* e)
+{
+	const MathLib::Vector3 c (e->getCenterOfGravity());
+	const unsigned nFaces (e->getNFaces());
+	for (unsigned j=0; j<nFaces; ++j)
+	{
+		MeshLib::Face const*const face (dynamic_cast<const MeshLib::Face*>(e->getFace(j)));
+		const MeshLib::Node x (*(face->getNode(1)));
```

Comment:
@rinkk shall I add your comment (about checking node 1) into the source so that we can avoid repating of the same question in the future?